# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/LayersTypes.h
# Commit: ea88b243717a
# Full Hash: ea88b243717a1dcdc2c02a57af70685959c5d649
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add D3D11 RenderCompositor for SWGL. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95797
# ==============================================================================

diff -r c68ab3e8a465 -r ea88b243717a gfx/layers/LayersTypes.h
--- a/gfx/layers/LayersTypes.h	Thu Nov 05 18:27:40 2020 +0000
+++ b/gfx/layers/LayersTypes.h	Fri Nov 06 00:00:47 2020 +0000
@@ -182,6 +182,7 @@
   DIRECT_COMPOSITION,
   CORE_ANIMATION,
   SOFTWARE,
+  D3D11,
   LAST
 };
 